code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -222,7 +222,7 @@
 				chambered = null
 				return
 
-		user.visible_message("<span class='danger'>*click*</span>")
+		audible_message("<span class='italics'>\The [src] clicks!</span>")
```

Comment:
I don't think it makes sense that you can hear a guns name just by the clicking.

Reply:
yeah, make it a visible message and potentially add something more generic as the blind_message

Reply:
OK, I'll change it. I just thought more descriptive was better in the heat of a firefight, even though it made no sense. I'm assuming this applies to all other heard clicks as well.

Reply:
keep the message as you changed it, just use the visible_message proc instead of audible_message